src/Microsoft.Extensions.Configuration.AzureAppConfiguration/IConfigurationHealthCheck.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
```

Comment:
Zhiyuan, can you help check what namespaces people use for their HealthCheck implementation? Do they put it in their own namespace or do they put it in the `Microsoft.Extensions.Diagnostics.HealthChecks` namespace?